apps/web/pages/team/[slug]/[type].tsx

```diff
@@ -171,6 +172,7 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
       hideBranding,
       timeZone,
     })),
+    descriptionAsSafeHTML: eventType.description ? parseAndSanitize(eventType.description) : null,
```

Comment:
Maybe we can simplify all these instances to `parseAndSanitize(eventType.description || "")`